Multi-Signal-Analyzer/capture_data.h

```diff
@@ -0,0 +1,28 @@
+/** -------------------------------------------------------------
+ * @file capture_data.h
+ * @brief Capture data module interface
+ *
+ * @author    Vinicius Rafael Marques de Carvalho vinicius.carvalho@edge.ufal.br
+ * @version   v1.0
+ * @date      28/11/2025
+ * @copyright
+ *  ------------------------------------------------------------*/
+
+#ifndef CAPTURE_DATA_H
+#define CAPTURE_DATA_H
+
+#include "pico/stdlib.h"
+
+/**
+ * @brief Initate the PIO and DMA
+ */
+int capture_init();
```

Comment:
Geralmente, adicionamos `void` na assinatura das funções que não recebem nenhum parametro. Pq em c, se não tiver nada, ainda é possível passar algo, mas se tiver `void` não é possível passar nada.
```suggestion
int capture_init(void);
```